extensions/amp-fit-text/1.0/storybook/Basic.js

```diff
@@ -34,7 +34,7 @@ export const _default = () => {
     <FitText
       minFontSize={minFontSize}
       maxFontSize={maxFontSize}
-      style={{display: 'block', border: '1px solid black', width, height}}
+      style={{border: '1px solid black', width, height}}
```

Comment:
Does it matter here if `display:block` is set? Does it go on a wrapper or a content element?

Reply:
Yes, unfortunately it depends also whether there is overflowing text. `display: block` here will undo centering when there is overflowing tex (which it was doing here already), but not when there is not overflowing text (which gets vertically centered within the intermediary `<div>`). I guess one option is we that the middle element could take the `display` value from the parent element so it will impact both? But it is not intuitive.

Reply:
I think we need an extra wrapper then. I think it'd be very natural for someone to want to style this element with `display: inline-block` or similar.

Reply:
To clarify: You want an extra wrapper so that setting `display` does not impact any form of internal centering?

Reply:
Would it not suffice to encourage folks to use `inline-flex`? Or is the intent to stabilize the unexpected centering behavior if they do use `inline-block`?

Reply:
That seems pretty fragile and leaks some implementation details. If we can isolate, that'd work better, imho. E.g. centering with flexbox is one way (and maybe the best way at this time). But there could be other possibilities as well.

Reply:
Added a wrapper as suggested.